CHANGELOG.md

```diff
@@ -6,6 +6,47 @@ in this file.
 Detailed changes to the Solar Forecast Arbiter Core python library can be found
 in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
 
+
+## [1.0beta5] - Unreleased
+
+### Added
+
+- All report plots are now created with Plotly by default. We found plotly to
+  currently contain functionality to meet the requirements of the Arbiter's
+  usecases. Plotly's more mature javascript library also makes it easier to
+  optimize reports for display in web browsers.
```

Comment:
```suggestion
- All report plots are now created with Plotly instead of Bokeh. Plotly's JSON specification and more mature javascript library makes it easier to optimize reports for display in web browsers.
```